lib/emoji.js

```diff
@@ -126,3 +127,25 @@ Emoji.search = function search(str) {
     };
   });
 }
+
+var emojiToCode = Object.keys(Emoji.emoji).reduce(function(h,k) {
+  return h[Emoji.emoji[k]] = k, h;
+}, {});
+
+/**
+ * unemojify a string (replace emoji with :emoji:)
+ * @param  {string} str
+ * @return {string}
+ */
+Emoji.unemojify = function unemojify(str) {
+  if (!str) return '';
+  var words = toArray(str);
```

Comment:
Does not work for complex emojis. Try for something like 👩‍👩‍👧‍👧. Thats why I used the lodash library.